firmware/controllers/algo/fuel/dfco.h

```diff
@@ -14,4 +14,5 @@ class DfcoController : public EngineModule {
 	bool m_isDfco = false;
```

Comment:
(not directly related to this PR)
is it time for dfco_controller.txt to get some LiveData logging?

Reply:
some day, probably 🙂